<launch>

    <group ns="/">
      <node 
        pkg="joystick2coms" 
        name="htry" 
        type="joy_node"
       />
    </group>

    <group ns="/">
      <node 
        pkg="joystick2coms" 
        name="can2dev" 
        type="can2dev"
	output="screen"
       />
    </group>

    <group ns="/">
      <node 
        pkg="joystick2coms" 
        name="canopen" 
        type="canopen.sh" 
        launch-prefix="terminator -x "
       />
    </group>
    
    <group ns="/">
      <node 
        pkg="rqt_reconfigure" 
        name="rqt_reconfigure" 
        type="rqt_reconfigure" 
       />
    </group>

    <group ns="/">
      <node 
        pkg="joystick2coms" 
        name="candump" 
        type="dumptest" 
        args="slcan0"
       />
    </group>
</launch>
